src/main/java/gov/nih/nci/evs/api/model/Synonym.java

```diff
@@ -434,4 +434,12 @@ public int compareTo(Synonym other) {
     return (source + type + name)
         .compareToIgnoreCase(other.getSource() + other.getType() + other.getName());
   }
+
+  /** Clear hidden. */
+  public void clearHidden() {
```

Comment:
It doesn't appear this method is getting called anywhere, so the normName is still appearing in the Synonym Properties.